server/data/mappers/ciagPlanMapper.ts

```diff
@@ -7,15 +7,19 @@ import SkillsValue from '../../enums/skillsValue'
 import PersonalInterestsValue from '../../enums/personalInterestsValue'
 import InPrisonWorkValue from '../../enums/inPrisonWorkValue'
 import InPrisonEducationValue from '../../enums/inPrisonEducationValue'
+import enumComparator from './enumComparator'
+import previousWorkExperienceDtoComparator from './previousWorkExperienceDtoComparator'
+import futureWorkInterestDtoComparator from './futureWorkInterestDtoComparator'
+import achievedQualificationDtoComparator from './achievedQualificationDtoComparator'
```

Comment:
I'm working in the same file atm, so the merge conflict is going to be fun!

In all seriousness, I'm wondering if this should be split up into different mappers, or we call it something like `inductionDtoToCiagPlanMapper`? Otherwise, it's growing quite large with the work I'm doing atm.